zulipterminal/model.py

```diff
@@ -775,7 +785,7 @@ def _handle_message_event(self, event: Event) -> None:
                 if (append_to_stream
                     and (len(self.narrow) == 1
                          or (len(self.narrow) == 2
-                             and self.narrow[1][1] == message['subject']))):
+                             and self.narrow[1][1].lower() == message['subject'].lower()))):
```

Comment:
It might be cleaner if we have a general helper of the kind `compare_lowercase` which can be used throughout our codebase to do lowercase comparisons.

Reply:
Thanks!